### PR TITLE
Convert examples to typescript

### PR DESCRIPTION
**Pull request by mharsat**

### CODE REVIEW

File: examples/storybook/stories/UploadMedia/Utils/ErrorBlocksRemovalApp.tsx [hunk 1]
```diff
@@ -5,18 +5,24 @@ import contentState from '../../../../../e2e/tests/fixtures/empty.json';
...
+  editor: RicosEditorType;
 
-export default class ErrorBlocksRemovalApp extends Component {
   constructor(props) {
     super(props);
     this.state = { isEditing: true, content: contentState };
   }
 
-  handleClick() {
+  async handleClick() {
```
Comment (lxgreen) [inline examples/storybook/stories/UploadMedia/Utils/ErrorBlocksRemovalApp.tsx:21]: huh.. FYI this word alone weights 10kB unzipped (just got flashback regarding recent discovery -- the `editor-common` package does not include the ES generators infra, so when i added `async` keyword, suddenly all editor-related bundles grew by 10kB)

**Review by lxgreen — APPROVED** [review]
reviewed the non-example code changes basically, looks good. assume the examples are also ok

**Comment by lxgreen** [comment]
i bet this PR adds about 30% TS coverage to the ricos

File: examples/storybook/stories/UploadMedia/Utils/ErrorBlocksRemovalApp.tsx (hunk 1 above)
Comment (mharsat) [reply examples/storybook/stories/UploadMedia/Utils/ErrorBlocksRemovalApp.tsx:21]: lol good thing we have the bundle analyzer

**Comment by mharsat** [comment]
Hope so!! 🚀🔥

**Pull request merged**